nuget/Microsoft.Windows.CsWinRT.Authoring.Transitive.targets

```diff
@@ -22,6 +22,7 @@
     <HostingAssets Include="$(MSBuildThisFileDirectory)..\build\native\WinRT.Host.runtimeconfig.json"/>
     <!-- Get the proper WinRT.Host.dll -->
     <HostingAssets Include="$(MSBuildThisFileDirectory)..\runtimes\win-$(_NormalizedPlatform)\native\WinRT.Host.dll" />
+    <HostingAssets Include="$(MSBuildThisFileDirectory)..\runtimes\win-$(_NormalizedPlatform)\native\en-us\WinRT.Host.dll.mui" />

```

Comment:
Whats the story here for localized ones if en-us is the main one copied  Is it up the component owners to pack the others based on which ones they want?  Same for below where you use en-us.

Reply:
Yeah I think automating minimally is better in this case so we don't always output many binaries in their project. The different culture/regions will be available in the nupkg, so they can distribute the whole lot if they want. But we will just handle en-us for them, for now. If we get feedback from folks that they prefer a different model we can adjust accordingly.